packages/frontend-2/components/project/page/automations/Header.vue

```diff
@@ -10,7 +10,7 @@
         placeholder="Search automations..."
         wrapper-classes="shrink-0"
         show-clear
-        :model-value="bind.modelValue.value"
+        v-bind="bind"
```

Comment:
Improved `useDebouncedTextInput` return to allow for way simpler/easier configuration of the FormTextInput component